submissions/Shrai-dev/html-css-popup/main.css

```diff
@@ -0,0 +1,179 @@
+    padding: 0;
+    box-sizing: border-box;
+    text-align: center;
+}
+
+img {
+    vertical-align: middle;
+}
+
+li {
+    list-style-type: none;
+}
+
+.nav__menu {
+    margin: 25px 0;
+    align-items: flex-end;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    height: 50px;
+}
+
+.nav__menu-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav__menu-item {
+    position: relative;
+    margin: 5px 20px;
+}
+
+.nav__menu-item > a:hover,
+.nav__menu-item > a:focus {
+    outline: thin solid #a3a2a2;
+    background-color: #c2c2c2;
+}
+
+a, 
+a:visited {
+    color: #6b6a6a;
+    text-decoration: none;
+}
+
+label > img:focus,
+label > img:hover,
+.popup__menu:hover,
+.popup__menu:focus {
+    cursor: pointer;
+    outline: thin solid #a3a2a2;
+    background-color: #c2c2c2;
+}
+
+.submenu__list, 
+.more__popup-menu,
+.more__popup {
+    display: none;
+}
+
+.popup__menu {
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    z-index: -1;
+    appearance: none;
+    outline: none;
+}
+
+#popup:checked ~ .submenu__list {
+    max-width: 350px;
+    width: 350px;
+    height: 500px;
+    position: absolute;
+    top: 45px;
+    right: -95px;
+    padding-top: 10px;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    overflow: auto;
+    outline: thin solid #c2c2c2;
+    box-shadow: 0 0 10px rgba(0,0,0,0.5);
+    font-size: 0.7em;
+    line-height: 1.5em;
+}
+
+.submenu__list-item a {
+    display: flex;
+    flex-direction: column;
+    flex-flow: row wrap;
+    justify-content: center;
+    text-decoration: none;
+    width: 100%;
+    height: 100%;
+    padding: 20%;
+}
+
+.submenu__list-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    box-sizing: border-box;
+    width: 29%;
+    height: 22%;
+}
+.submenu__list-item:hover,
+.submenu__list-item:focus {
+    outline: thin solid #c2c2c2;
+    background-color: #c2c2c2;
+}
+.submenu__list-item img {
+    max-width: 90%;
+    max-height: 60%;
+    margin-bottom: 10%;
+}
+
+.more__list {
+    margin-top: 10px;
+    text-align: center;
+    width: 100%;
+    height: 40px;
+    vertical-align: middle;
+    color: #6b6a6a;
+}
+
+.more__list:focus {
+    outline: thin solid #c2c2c2;
+    background-color: transparent;
+}
+
+#more-popup:checked ~ .more__popup-menu {
+    max-width: 100%;
+    position: relative;
+    padding-top: 5px;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    line-height: 1.5em;
+}
+
+#more-popup:checked ~ .more__popup-menu::before {
+    content: "";
+    width: 100%;
+    border-top: 2px solid #c4c4c4;
+    padding: 10px 0;
+}
+
+#more-popup:checked ~ .more__submenu, 
+#more-popup:checked {
+    display: none;
```

Comment:
Try to use just classes white you're styling

Reply:
Done.